scripts/xe-reset-networking

```diff
@@ -53,7 +53,7 @@ if __name__ == "__main__":
 	parser.add_option("-m", "--master", help="Master's address", dest="address", default=None)
 	parser.add_option("--device", help="Device name of new management interface", dest="device", default=None)
 	parser.add_option("--mode", help='IP configuration mode for new management interface: "dhcp" or "static" (default is dhcp)', dest="mode", default="dhcp")
-       parser.add_option("--vlan", help="vlanID for new management interface to be on vlan network", dest="vlan", default='')
+        parser.add_option("--vlan", help="vlanID for new management interface to be on vlan network", dest="vlan", default='')
```

Comment:
Indentation looks weird here but thats actually correct.
After the merge of previous commit https://github.com/xenserver/xen-api/pull/259/commits/8e095fc99899a879c37eddcdc04c648d33ac6601
The changes appeared on branch trunk-car-1841 was like:
` parser.add_option("--mode", help='IP configuration mode for new management interface: "dhcp" or "static" (default is dhcp)', dest="mode", default="dhcp")
parser.add_option("--vlan", help="vlanID for new management interface to be on vlan network", dest="vlan", default='')
 parser.add_option("--ip", help="IP address for new management interface", dest="ip", default='')`